ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -1313,6 +1313,52 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: baremetal-agent-ipv4-static-amd-mixarch-f14
```

Comment:
Hi @zniu1011, if I understand correctly, this should be considered a compact cluster (3-masters only) with a day2 scale up operation. Should we explicitly clarify it as in the other profiles?

Reply:
Ok

Reply:
Yeah let's, reduce some chars maybe? Not sure what we usually do in these cases, but `compact` looks important to state as the 3 master nodes will also get the worker role (regardless of the scale up)

Reply:
there are some cases using `disc` replace `disconnected`

Reply:
Thanks @lwan-wanglin, let me try 'disc' instead of 'disconnected'